client/my-sites/checkout/cart/cart-item.jsx

```diff
@@ -61,7 +61,8 @@ export class CartItem extends React.Component {
 			return this.getDomainPlanPrice( cartItem );
 		}
 
-		const cost = cartItem.cost * cartItem.volume;
+		const unformattedCost = cartItem.cost * cartItem.volume;
+		const cost = unformattedCost.toFixed( getCurrencyDefaults( cartItem.currency ).precision );
```

Comment:
That's weird. The unit tests for this component are failing with `TypeError: (0 , _formatCurrency.getCurrencyDefaults) is not a function`, but I see that function exported by the package and used in other places. 🤔  

It sounds to me like something wrong with the test. However! I was thinking, there's really no need for this calculation at all. We can do things like [new checkout does them](https://github.com/Automattic/wp-calypso/blob/0799ab42a1d41122373468cbd33d6dda20be5291/client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx#L884), and use `cartItem.item_subtotal_display` which should include the pre-calculated, localized price with its currency symbol.

Reply:
Sure thing, done! :) 

<img width="392" alt="Screenshot 2021-08-04 at 12 12 48" src="https://user-images.githubusercontent.com/43215253/128171469-2d1fbac1-5956-491f-8dac-086b0bf29941.png">